metricbeat/module/kubernetes/state_deployment/_meta/data.json

```diff
@@ -0,0 +1,29 @@
+{
+  "@timestamp": "2017-05-04T17:47:01.122Z",
+  "beat": {
+    "hostname": "X1",
+    "name": "X1",
+    "version": "6.0.0-alpha1"
+  },
+  "kubernetes": {
+    "deployment": {
+      "name": "jumpy-owl-redis",
+      "paused": false,
+      "replicas": {
+        "available": 0,
+        "desired": 1,
+        "unavailable": 1,
+        "updated": 1
+      }
+    },
+    "namespace": "default",
+    "state_deployment": {}
```

Comment:
This should not exist. Probably you should change the namespace of the metricset instead of putting it under the module namespace.